analysis/src/python/data_collection/hyperskill/api/projects.py

```diff
@@ -0,0 +1,45 @@
+from dataclasses import dataclass, field
+from typing import List, Optional, Dict
+
+from analysis.src.python.data_collection.api.platform_entities import Object, RequestParams, Response
+
+
+@dataclass
+class ProjectsRequestParams(RequestParams):
+    pass
+
+
+@dataclass
+class Project(Object):
+    id: int
+    title: str
+    use_ide: bool
+    environment: str
+    description: str
+    is_beta: bool
+    is_template_based: bool
+    results: str
+    stages_count: int
+    n_first_prerequisites: int
+    n_last_prerequisites: int
+    language: str
+    is_deprecated: bool
+    progress_id: str
+    readiness: int
+    lesson_stepik_id: Optional[int]
+    preview_step: Optional[int] = None
+    ide_files: Optional[str] = None
+    stages_ids: List[int] = field(default_factory=list)
+    url: str = field(init=False)
+    tracks: Dict[str, Dict[str, str]] = field(default_factory=dict)
+
+    def __post_init__(self):
+        self.url = f'https://hyperskill.org/projects/{self.id}'
```

Comment:
Please, create a base class or something like this with the base URL and use it everywhere for hyperskill queries classes